### ISSUE TITLE
EMAGE: Traning of vqvae_lower_foot is quite unstable

### ISSUE THREAD

**Issue by weihaosky** [PantoMatrix/PantoMatrix#149]
Hi, the training is vqvae_lower_foot is quite unstable, as

![rec_cnn_vqvae_lower_foot_30_0607_110753](https://github.com/PantoMatrix/PantoMatrix/assets/23240241/cac2b1b9-8656-4ba1-812f-1138f6dbe94f)

![com_cnn_vqvae_lower_foot_30_0607_110753](https://github.com/PantoMatrix/PantoMatrix/assets/23240241/0d761052-880b-4de0-b0b7-b919d01cc6b9)

Is this normal? 

Thanks

**Comment by H-Liu1997** [comment]
it is possible with CNN based encoder, it will be better with SKCNN or gradnorm < 1.0, but my loss is stable, have you modified something? 
thanks!

**Comment by weihaosky** [comment]
> it is possible with CNN based encoder, it will be better with SKCNN or gradnorm < 1.0, but my loss is stable, have you modified something? thanks!

I did not modify anything.

**Comment by WISHF-Amoy** [comment]
I also find this problem, and just edit the train epochs as the description in supplementary material, the training loss is really strange for the vqvae lower foot, and only for this part, others are really good. By the way, if this unstable loss lead to bad performance in the final EMAGE retrained, because I got FGD much higher than reported in paper.
